### PR TITLE
OCPBUGS#15328: Update notes

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-15328

Link to docs preview: https://61747--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

QE review: @geliu2016 
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ This procedure assumes that you copied the `backup` directory containing the etc
 +
 [NOTE]
 ====
-It is not required to manually stop the pods on the recovery host. The recovery script will stop the pods on the recovery host.
+Stop the static pods on the control plane nodes other than the recovery host.
```
Comment (skopacz1) [inline modules/dr-restoring-cluster-state.adoc:56]: At this point the admonition is worded very similarly to the step it modifies. I would consider either of the following alternatives:

1. Replace line 52 with this sentence and get rid of this admonition.
2. Reword this note to something like the following:
```suggestion
You do not need to stop the static pods on the recovery host.
```

File: modules/dr-restoring-cluster-state.adoc [hunk 2]
```diff
@@ -206,7 +206,7 @@ $ sudo systemctl restart kubelet.service
 +
 [NOTE]
 ====
-Clusters with no worker nodes, such as single-node clusters or clusters consisting of three schedulable control plane nodes, will not have any pending CSRs to approve. In those scenarios, you can skip this step.
+Clusters with no worker nodes, such as single-node clusters or clusters consisting of three schedulable control plane nodes, will not have any pending CSRs to approve. you can skip all the commands listed in this step.
```
Comment (skopacz1) [inline modules/dr-restoring-cluster-state.adoc:209]: Nit and capitalization:

```suggestion
Clusters with no worker nodes, such as single-node clusters or clusters consisting of three schedulable control plane nodes, will not have any pending CSRs to approve. 

You can skip all the commands listed in this step if there are no worker nodes.
```

File: modules/dr-restoring-cluster-state.adoc [hunk 3]
```diff
@@ -290,20 +290,15 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
...
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
-+
-[NOTE]
-====
-Perform the following step only if you are using `OVNKubernetes` network plugin.
-====
 
-. Delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
+. If you are using the `OVNKubernetes` network plugin, delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
```
Comment (skopacz1) [inline modules/dr-restoring-cluster-state.adoc:294]: Nice, this is a much better way to document the conditional step compared to having that note above.

File: modules/dr-restoring-cluster-state.adoc [hunk 4]
```diff
@@ -290,20 +290,15 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
...
 +
 [source,terminal]
 ----
 $ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
 ----
 
-. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed with the next step.
+. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed restarting the Open Virtual Network (OVN) Kubernetes pods on all the hosts in the next step.
```
Comment (skopacz1) [inline modules/dr-restoring-cluster-state.adoc:301]:
```suggestion
. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed to restart the Open Virtual Network (OVN) Kubernetes pods on all of the hosts in the next step.
```

Also I'm not familiar with OVN, should "OVN-Kubernetes" in the first sentence be in backticks?

**Review by skopacz1 — COMMENTED** [review]
Left a few comments, otherwise looks good! Please wait until @adellape approves of my review before implementing any feedback, thanks.

File: modules/dr-restoring-cluster-state.adoc (hunk 4 above)
Comment (adellape) [reply modules/dr-restoring-cluster-state.adoc:301]: Typically it is not, across the repo. However it seems like the spelled out "Open Virtual Network (OVN) Kubernetes" should be in the first sentence if that's the first appearance in this module, versus it being in a subsequent instance.

**Review by adellape — COMMENTED** [review]
@skopacz1 @xenolinux Peer review comments LGTM 👍

**Comment by abhatt-rh** [comment]
Hi @xenolinux The cherry-picks to 4.11 and 4.10 failed, so they need to be manually cherry-picked. Leaving the merge labels on for now until the manual PRs are created/merged.

**Pull request merged**